src/app/components/table-row-option-menu.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="dropdown">
+  <button
+    class="btn dropdown-toggle disable-caret"
+    type="button"
+    id="dropdownMenuButton-{{ rowIndex }}"
```

Comment:
This is just a global id to track bootstrap right? I wonder if we should just take the opportunity to replace the dropdown logic with https://material.angular.io/cdk/overlay/overview